<launch>
  <node name="rosserial_python" pkg="rosserial_python" type="serial_node.py" respawn="true" args="/dev/ttyACM0" output="screen"/>
  <node name="ctrl" pkg="differential_drive" type="KeyboardControl"/>
  <node name="key" pkg="differential_drive" type="KeyboardEvent"/>
</launch>
